src/saml2/sigver.py

```diff
@@ -406,18 +396,10 @@ def active_cert(key):
     """
     try:
         cert_str = pem_format(key)
-        try:
-            certificate = importKey(cert_str)
-            not_before = to_time(str(certificate.get_not_before()))
-            not_after = to_time(str(certificate.get_not_after()))
-            assert not_before < utc_now()
-            assert not_after > utc_now()
-            return True
-        except:
-            cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert_str)
-            assert cert.has_expired() == 0
-            assert not OpenSSLWrapper().certificate_not_valid_yet(cert)
-            return True
+        cert = crypto.load_certificate(crypto.FILETYPE_PEM, cert_str)
+        assert cert.has_expired() == 0
```

Comment:
I'm simply replacing an existing assertion. I expect @rohe already knows that about this code.